src/runtime/internal/Component.ts

```diff
@@ -103,7 +104,7 @@ function make_dirty(component, i) {
 	component.$$.dirty[(i / 31) | 0] |= (1 << (i % 31));
 }
 
-export function init(component, options, instance, create_fragment, not_equal, props, dirty = [-1]) {
+export function init(component, options, instance, create_fragment, not_equal, props, append_styles, dirty = [-1]) {
```

Comment:
Strictly speaking it is `append_styles = null`

Reply:
we can safe additional 10 bytes by omitting `=null`

Reply:
Shouldn't consider backward compatibility ?